mlrun/package/packagers/pandas_packagers.py

```diff
@@ -94,27 +193,452 @@ class _CSVFormatter(_Formatter):
+            if obj.columns[0] == "Unnamed: 0":
+                obj.drop(columns=["Unnamed: 0"], inplace=True)
+            # Unflatten the dataframe:
+            obj = cls._unflatten_dataframe(dataframe=obj, **unflatten_kwargs)
+
+        return obj
+
+
+class _H5Formatter(_Formatter):
+    """
+    A static class for managing pandas h5 files.
+    """
+
+    @classmethod
+    def to(
+        cls, obj: pd.DataFrame, file_path: str, flatten: bool = True, **to_kwargs
+    ) -> dict:
+        """
+        Save the given dataframe to the h5 file path given.
+
+        :param obj:       The dataframe to save.
+        :param file_path: The file to save to.
+        :param flatten:   Ignored for h5 format.
+        :param to_kwargs: Additional keyword arguments to pass to the `to_hdf` function.
+
+        :return A dictionary of keyword arguments for reading the dataframe from file.
+        """
+        # If user didn't provide a key for the dataframe, use default key 'table':
+        key = to_kwargs.pop("key", "table")
+
+        # Write to h5:
+        obj.to_hdf(path_or_buf=file_path, key=key, **to_kwargs)
+
+        return {"key": key}
+
+    @classmethod
+    def read(
+        cls, file_path: str, unflatten_kwargs: dict = None, **read_kwargs
+    ) -> pd.DataFrame:
+        """
+        Read the dataframe from the given h5 file path.
+
+        :param file_path:        The file to read the dataframe from.
+        :param unflatten_kwargs: Ignored for h5 format.
+        :param read_kwargs:      Additional keyword arguments to pass to the `read_hdf` function.
+
+        :return: The loaded dataframe.
+        """
+        return pd.read_hdf(path_or_buf=file_path, **read_kwargs)
+
+
+class _XMLFormatter(_Formatter):
+    """
+    A static class for managing pandas xml files.
+    """
+
+    @classmethod
+    def to(
+        cls, obj: pd.DataFrame, file_path: str, flatten: bool = True, **to_kwargs
+    ) -> dict:
+        """
+        Save the given dataframe to the xml file path given.
+
+        :param obj:       The dataframe to save.
+        :param file_path: The file to save to.
+        :param flatten:   Whether to flatten the dataframe before saving. For some formats it is mandatory to enable
+                          flattening, otherwise saving and loading the dataframe will cause unexpected behavior
+                          especially in case it is multi-level or multi-index. Default to True.
+        :param to_kwargs: Additional keyword arguments to pass to the `to_xml` function.
+
+        :return A dictionary of keyword arguments for reading the dataframe from file.
+        """
+        # Get the parser (if not provided, try to use `lxml`, otherwise `etree`):
+        parser = to_kwargs.pop("parser", None)
+        if parser is None:
+            try:
+                importlib.import_module("lxml")
+                parser = "lxml"
+            except ModuleNotFoundError:
+                parser = "etree"
```

Comment:
should work here as well
```suggestion
parser = "lxml" if "lxml" in sys.modules else "etree"
```

Reply:
see previous comment